src/utils/actionTypes.js

```diff
@@ -4,21 +4,18 @@
  * If the current state is undefined, you must return the initial state.
  * Do not reference these action types directly in your code.
  */
+
+const randomString = () =>
+  Math.random()
+    .toString(36)
+    .substring(7)
+    .split('')
+    .join('.')
+
 const ActionTypes = {
-  INIT:
-    '@@redux/INIT' +
-    Math.random()
-      .toString(36)
-      .substring(7)
-      .split('')
-      .join('.'),
-  REPLACE:
-    '@@redux/REPLACE' +
-    Math.random()
-      .toString(36)
-      .substring(7)
-      .split('')
-      .join('.')
+  INIT: `@@redux/INIT${randomString()}`,
+  REPLACE: `@@redux/REPLACE${randomString()}`,
+  PROBE_UNKNOWN_ACTION: () => `@@redux/PROBE_UNKNOWN_ACTION_${randomString()}`
```

Comment:
This type has a different format.
It contains an extra `_` before `randomString`.

Reply:
@theJian `PROBE_UNKNOWN_ACTION ` already contains '_' before `randomString`.

Reply:
I see. Maybe it's a typo.
But I think we can remove it as those types are grouped together now.

Reply:
Ok, I understand what you mean.  But I think we can't delete this `_` . 
Because it may have been intentionally added by previous [versions](https://github.com/reactjs/redux/blob/master/src/combineReducers.js#L81). 

PS: I tried to delete it, test can still pass , So I'm not sure if I can delete it . 🤔